testpilot/frontend/static/images/share-email.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Oh, I see what happened here! These images belong in `testpilot/frontend/static-src`, not `testpilot/frontend/static`. 

The [error on CircleCI](https://circleci.com/gh/mozilla/testpilot/629) happens because gulp via `npm build` deletes the contents of `testpilot/frontend/static` before doing it's thing - which includes copying over `testpilot/frontend/static-src/images`. Then, Django's `collectstatic` stumbled over the deleted images referenced in the CSS.